users/users.go

```diff
@@ -66,14 +71,23 @@ func isEmail(email string) bool {
 		return false
 	}
 
-	user := email[:at]
+	local := email[:at]
 	host := email[at+1:]
```

Comment:
Split would simplify all here:
```
    es := strings.Split(email, "@")
    local, host := es[0], es[1]
```

Reply:
I've tried to make minimum changes

Reply:
It is out of issue scope

Reply:
I understand and I know that you didn't write this code but it would be the perfect moment to fix this :)